examples/hcp-aks-demo/main.tf

```diff
@@ -0,0 +1,114 @@
+data "azurerm_subscription" "current" {}
+
+resource "azurerm_resource_group" "rg" {
+  name     = "${var.cluster_id}-gid"
+  location = var.network_region
+}
+
+resource "azurerm_route_table" "rt" {
+  name                = "${var.cluster_id}-rt"
+  resource_group_name = azurerm_resource_group.rg.name
+  location            = azurerm_resource_group.rg.location
+}
+
+resource "azurerm_network_security_group" "nsg" {
+  name                = "${var.cluster_id}-nsg"
+  resource_group_name = azurerm_resource_group.rg.name
+  location            = azurerm_resource_group.rg.location
+}
+
+module "network" {
+  source              = "Azure/vnet/azurerm"
+  resource_group_name = azurerm_resource_group.rg.name
+  address_space       = var.vnet_cidrs
+  subnet_prefixes     = values(var.vnet_subnets)
+  subnet_names        = keys(var.vnet_subnets)
+  vnet_name           = "${var.cluster_id}-vnet"
+
+  # Every subnet will share a single route table
+  route_tables_ids = { for i, subnet in keys(var.vnet_subnets) : subnet => azurerm_route_table.rt.id }
+
+  # Every subnet will share a single network security group
+  nsg_ids = { for i, subnet in keys(var.vnet_subnets) : subnet => azurerm_network_security_group.nsg.id }
+
+  depends_on = [azurerm_resource_group.rg]
+}
+
+resource "hcp_hvn" "hvn" {
+  hvn_id         = var.hvn_id
+  cloud_provider = "azure"
+  region         = var.hvn_region
+  cidr_block     = var.hvn_cidr_block
+}
+
+module "hcp_peering" {
+  #source  = "hashicorp/hcp-consul/azurerm"
+  #version = "~> X.X.X"
+  # TODO: Revert to above once this is published
+  source  = "../.."
```

Comment:
Is this necessary or can we revert to a publish version now?

Reply:
I'm actually not sure. I would think that the client module won't be published until this is merged so there is not valid version to stub in here yet?